src/datapump/sync/sync.py

```diff
@@ -458,19 +458,24 @@ def get_latest_release(self) -> Tuple[str, List[str]]:
             LOGGER.info(
                 f"Found {version_tiles} RADD tiles in gs://{self.source_bucket}/{version_prefix}"
             )
-            if version_tiles > self.number_of_tiles:
-                raise Exception(
-                    f"Found {version_tiles} TIFFs in latest {self.dataset_name} GCS folder, which is "
-                    f"greater than the expected {self.number_of_tiles}. "
-                    "If the extent has grown, update NUMBER_OF_TILES value."
-                )
-            elif version_tiles == self.number_of_tiles:
+            # Why is self.number_of_tiles a list? Because it seems that it
```

Comment:
I know this solves the current issue so for future discussion: how about specifying a range of acceptable number of tiles so it doesn't break on reasonable number not captured by the list? That with logging a warning when it happens could avoid the disruption.

Reply:
Thank you for the ideas, folks!
It's tricky to figure out what exactly to do for a few reasons, and some assumptions need to be re-examined. I think I'll merge this to get things going again, but also start a discussion in the Slack channel.